app/main/__init__.py

```diff
@@ -1,6 +1,9 @@
 from flask import Blueprint
+from .helpers.auth import requires_auth
 
 main = Blueprint('main', __name__)
 
+main.before_request(requires_auth)
```

Comment:
@TheDoubleK Figured it out this morning, have a look at 6e370fc and see if that meets your needs.